schemas/clinvar-study.tables.json

```diff
@@ -0,0 +1,3068 @@
+          },
+          {
+            "name": "geno_type",
+            "datatype": "string"
+          },
+          {
+            "name": "review_status",
+            "datatype": "string"
+          },
+          {
+            "name": "rcv_list",
+            "datatype": "string"
+          },
+          {
+            "name": "clinical_assertion_list",
+            "datatype": "string"
+          },
+          {
+            "name": "trait_mapping_list",
+            "datatype": "string"
+          },
+          {
+            "name": "deleted_scv_list",
+            "datatype": "string"
+          },
+          {
+            "name": "general_citations",
+            "datatype": "string",
+            "array_of": true
+          }
+        ]
+      },
+      {
+        "name": "type_allele",
+        "columns": [
+          {
+            "name": "persistent_id",
+            "datatype": "string"
+          },
+          {
+            "name": "gene_list",
+            "datatype": "string"
+          },
+          {
+            "name": "name",
+            "datatype": "string"
+          },
+          {
+            "name": "variant_type",
+            "datatype": "string"
+          },
+          {
+            "name": "location",
+            "datatype": "string"
+          },
+          {
+            "name": "other_name_list",
+            "datatype": "string"
+          },
+          {
+            "name": "protein_change",
+            "datatype": "string",
+            "array_of": true
+          },
+          {
+            "name": "hgvs_list",
+            "datatype": "string"
+          },
+          {
+            "name": "interpretations",
+            "datatype": "string"
+          },
+          {
+            "name": "xref_list",
+            "datatype": "string"
+          },
+          {
+            "name": "comment",
+            "datatype": "string"
+          },
+          {
+            "name": "functional_consequence",
+            "datatype": "string"
+          },
+          {
+            "name": "allele_frequency_list",
+            "datatype": "string"
+          },
+          {
+            "name": "global_minor_allele_frequency",
+            "datatype": "string"
+          },
+          {
+            "name": "allele_id",
+            "datatype": "integer"
+          },
+          {
+            "name": "variation_id",
+            "datatype": "integer"
+          }
+        ]
+      },
+      {
+        "name": "gene_list",
+        "columns": [
+          {
+            "name": "persistent_id",
+            "datatype": "string"
+          },
+          {
+            "name": "gene",
+            "datatype": "string",
+            "array_of": true
+          },
+          {
+            "name": "gene_count",
+            "datatype": "integer"
+          }
+        ]
+      },
+      {
+        "name": "gene",
+        "columns": [
+          {
+            "name": "persistent_id",
```

Comment:
Do any of the other columns in here make sense to use as `persistent_id`, or will this be randomly-generated?